ecommerce/frontend/src/components/product_card/ProductCard.js

```diff
@@ -7,11 +7,11 @@ import './ProductCard.less'
 const { Meta } = Card
 
 export const ProductCard = ({ product, onAddToCart, width = 350 }) => {
-    const { title, rating, price, currency, imageUrl, productId } = product
+    const { title, rating, price, currency, imageUrl, id } = product
```

Comment:
If the database sends `image_url` rather than `imageUrl`, i believe you also need to change this.